build.gradle

```diff
@@ -185,24 +258,24 @@ subprojects {
                     version project.version
                 }
             }
-        }.withXml {
-            project.configurations[JavaPlugin.RUNTIME_CONFIGURATION_NAME].allDependencies.findAll {
```

Comment:
Task of `pom` is still for local development (and validation in `ci/validate.sh`). `pom` directive in `publishing` is for publishing. They share the same code (`pomXmlModifier`) to generate the same `<dependencies>` section.